docs/core/diagnostics/runtime-contention-events.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Thread Contention Events"
+description: See ETW events that collect information specific to the monitor lock contentions.
```

Comment:
Need more keywords here and things to help search. Words like .NET, diagnostic, runtime, etc.